app/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:11.4.0
 WORKDIR /app
 COPY package.json .
 COPY yarn.lock .
-RUN yarn install
+RUN yarn install --network-timeout 1000000
```

Comment:
I assume this is on purpose but just checkin haha

Reply:
Yea. I was getting timeout issues. This Dockerfile is local, so should be ok